osu.Game.Rulesets.Osu/Objects/Drawables/DrawableHitCircle.cs

```diff
@@ -42,7 +42,7 @@ public DrawableHitCircle(OsuHitObject h) : base(h)
 
                         Judgement.PositionOffset = Vector2.Zero; //todo: set to correct value
                         UpdateJudgement(true);
-                        return true;
+                        return !(Parent is DrawableSlider);

```

Comment:
What if you enter the slider in an already down state? What if that down state was triggered before the slider was even alive? Not gonna work.